.yarn/versions/51d71892.yml

```diff
@@ -0,0 +1,4 @@
+declined:
+  - primitives
+  - ssr-testing
+  - "@radix-ui/react-hover-card"
```

Comment:
Unsure on release strategy, ran a `version check --interactive` to sidestep the pre-push

Reply:
That's the right command to run yep. In this case you just need to select patch for hover card so it gets released properly as 0.0.1 next time we do a release.

Reply:
Looks like this is still awaiting the update to move `@radix-ui/react-hover-card` to patch.

Happy to have a quick call to show you how to do that if it's not clear! LMK 🙂

Reply:
Should be ok now 🙂